src/ui/webeiddialog.cpp

```diff
@@ -144,6 +170,11 @@ WebEidDialog::WebEidDialog(QWidget* parent) : WebEidUI(parent), ui(new Private)
         ui->pinTimeRemaining->setText(
             tr("Time remaining: <b>%1</b>").arg(ui->pinEntryTimeoutProgressBar->maximum() - value));
     });
+
+    connect(this, &WebEidDialog::languageChange, this, [this]{
+        ui->pinInputCertificateInfo->languageChange();
+        QTimer::singleShot(0, this, &WebEidDialog::resizeHeight);
```

Comment:
The `QTimer` may not be needed, see comment below at `WebEidDialog::event()`.

Reply:
It is needed to schedule relayout later. Some elements are connected later than this signal.